packages/hardhat-core/src/internal/hardhat-network/stack-traces/vm-debug-tracer.ts

```diff
@@ -22,7 +22,23 @@ export class VMDebugTracer {
     this._afterTxHandler = this._afterTxHandler.bind(this);
   }
 
-  public enableTracing(config: RpcDebugTracingConfig) {
+  /**
+   * Run the `cb` callback and trace its execution
+   */
+  public async trace(
+    cb: () => Promise<void>,
```

Comment:
Maybe it's better to call this: `action` instead of `callback` for two reasons.
1. We already have a function `_runInBlockContext` that does similar stuff and there the parameter is called `action`.
2. `Callback` implies fire and forget callback syntax.

Reply:
Good point, fixed!